packages/office-ui-fabric-react/src/components/Divider/examples/VerticalDivider.Custom.Example.tsx

```diff
@@ -31,27 +31,26 @@ const getExampleClassNames = memoizeFunction(
   },
 );
 
-export class VerticalDividerCustomExample extends React.Component<any, any> {
-  public render(): JSX.Element {
-    const exampleClassNames = getExampleClassNames();
-    return (
-      <div className={exampleClassNames.wrapper}>
-        <p className={exampleClassNames.text}> Some text before the divider. </p>
-        <VerticalDivider
-          styles={{
-            wrapper: {
-              height: 40,
-              backgroundColor: '#F4F4F4',
-              padding: 0,
-            },
-            divider: {
-              height: 28,
-              backgroundColor: 'pink',
-            },
-          }}
-        />
-        <p className={exampleClassNames.text}>Some text after the divider. </p>
-      </div>
-    );
-  }
-}
+export const VerticalDividerCustomExample = () => {
+  const exampleClassNames = getExampleClassNames();
```

Comment:
Same here about getting rid of the function

Reply:
Probably fine to leave, as calling getTheme would get theme values at render, rather than at bundle parsing time.

Reply:
We really should change this--running an unnecessary calculation every render isn't a good pattern. 

Regarding calling `getTheme` at parse time, lots of other examples do this currently. Maybe not ideal, but if we want to change it, that should probably be approached all up.

If the styles must stay inside a function call, at least wrap the call in `useConst`:
`const exampleClassNames = useConst(() => getExampleClassNames());`